src/tests/ftest/util/server_utils.py

```diff
@@ -41,8 +41,8 @@
 
 SESSIONS = {}
 
-DEFAULT_FILE = "src/tests/ftest/data/daos_server_baseline.yaml"
-AVOCADO_FILE = "src/tests/ftest/data/daos_avocado_test.yaml"
+DEFAULT_FILE = "lib/daos/TESTING/ftest/data/daos_server_baseline.yaml"
```

Comment:
I'd suggest using relative paths here:
1) git move the `src/tests/ftest/data` directory to `src/tests/ftest/util/data`
2) Use these definitions for the constants:
```
DEFAULT_FILE = "./data/daos_server_baseline.yaml"
AVOCADO_FILE = "./data/daos_avocado_test.yaml"
```